### PR TITLE
Add netcup

### LINKED ISSUE #6151: netcup
**Brand Name:**netcup
**Website:**netcup.de
**Alexa rank:**100,515
  

**Official resources for icon and color:**https://www.netcup.de/static/assets/images/favicons/favicon.svg

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
netcup

**Issue:** closes #6151 

**Similarweb rank:** 279,478. See https://www.similarweb.com/website/netcup.de

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from favicon https://www.netcup.de/static/assets/images/favicons/favicon.svg
Color: Taken from logo `#056473`

### CODE REVIEW

**Comment by adamrusted** [comment]
Looks like we'll need permission @PratyushJayachandran:
https://www.netcup-wiki.de/wiki/Werbemittel#rechtliche_Hinweise

**Comment by PratyushJayachandran** [comment]
> Looks like we'll need permission @PratyushJayachandran: https://www.netcup-wiki.de/wiki/Werbemittel#rechtliche_Hinweise image

Awaiting reply

**Comment by PratyushJayachandran** [comment]
@adamrusted 
image

I changed name to lowercase. Will stick with favicon for readability

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I'm seeing [hairline differences](https://github.com/simple-icons/simple-icons/assets/15157491/8360d855-b337-4282-8241-c79446dbf546) in the corners, @PratyushJayachandran, which are likely just a precision issue.

We should also just use the website, or guidelines URL, for the source.

**Comment by PratyushJayachandran** [comment]
> I'm seeing [hairline differences](https://github.com/simple-icons/simple-icons/assets/15157491/8360d855-b337-4282-8241-c79446dbf546) in the corners, @PratyushJayachandran, which are likely just a precision issue.
> 
> We should also just use the website, or guidelines URL, for the source.

I redid the icon and saw no diff, so I bumped up the precision to 4.
As for the source, since it is not present on front page I used the exact link to the icon contributed. Guidelines page doesn't have that icon and I already told the representative that I would be using this source and sent him the PR too.

File: icons/netcup.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>netcup</title><path d="M18.75 0H5.25C2.3505 0 0 2.3505 0 5.25v13.5C0 21.6495 2.3505 24 5.25 24h13.5c2.8995 0 5.25-2.3505 5.25-5.25V5.25C24 2.3505 21.6495 0 18.75 0zm.0445 18.8988h-4.6661V8.581c0-.514-.2966-.7853-.8878-.7853h-3.367v11.1031H6.6826V7.7958h-1.477V5.1011h9.4827c1.7441 0 2.6296.9085 2.6296 2.7549v8.3507h1.4767v2.692z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/netcup.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>netcup</title><path d="M5.25 0A5.239 5.239 0 0 0 0 5.25v13.5A5.239 5.239 0 0 0 5.25 24h13.5A5.239 5.239 0 0 0 24 18.75V5.25A5.239 5.239 0 0 0 18.75 0H5.25zm-.045 5.102h9.482c1.745 0 2.631.907 2.631 2.753v8.352h1.477v2.691h-4.666V8.58c0-.514-.298-.785-.889-.785H9.873v11.103H6.682V7.795H5.205V5.102z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I'm still seeing those same differences, unfortunately, @PratyushJayachandran 🤔

In the interests of getting this one merged - 'cause I'm on a rampage this week, trying to close out as many stale issues as I can - I've given it a try myself.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**